manifests/plugin/foreman_gutterball_config.pp

```diff
@@ -0,0 +1,28 @@
+# foreman_gutterball config
+class katello::plugin::foreman_gutterball_config(
+    $foreman_user  = 'foreman',
+    $foreman_group = 'foreman',
+    $foreman_plugins_dir = '/usr/share/foreman/plugins',
+    $foreman_gutterball_gem_config = undef
+  ){
+  $config_file = "${foreman_plugins_dir}/foreman_gutterball.yaml"
+
+
+  concat{ $config_file: 
```

Comment:
I am not following why all this concatenation needs to occur to deploy this settings file?